### PR TITLE
0x05a-Platform-Overview - line 1-30

### PR DESCRIPTION
**Pull request by AnnaSzk**
Smaller linguistic changes and a comment asking to correct the unclear sentence

### CODE REVIEW

**Comment by Sjord** [comment]
I don't think these are improvements at all. Where you insert or remove "the" or "a" it was already correct, and becomes incorrect after your change.

File: Document/0x05a-Platform-Overview.md [hunk 1]
```diff
@@ -11,21 +11,21 @@ Visit the official [Android developer documentation website](https://developer.a
...
+Android is a Linux-based open-source platform initially developed by Google as a mobile operating system (OS) solution. Today, the platform is a foundation for a wide variety of modern technology such as mobile phones and tablets, wearable tech, and other "smart" devices like TVs. Typical Android builds ship with a range of pre-installed ("stock") apps and support installation of third-party apps through Google Play store and other marketplaces.
 
-The software stack of Android is composed of several different layers. Each layer defines certain behavior and offers specific services. Further, these layers interact with OS portions above, below, and other instances paralleling the respective layer’s position within the system architecture.
+The software stack of Android is composed of several different layers. Each layer defines particular behavior and offers specific services. Further, these layers interact with OS portions above, below, and other instances paralleling the respective layer’s position within the system architecture. - ASZ - the phrase is unclear and requires re-phrasing
```
Comment (Sjord) [inline Document/0x05a-Platform-Overview.md]: >  - ASZ - the phrase is unclear and requires re-phrasing

First, don't put comments like this in the text. Second, the sentence makes more sense in the context of the image shown below it.

**Comment by Sjord** [comment]
I am inclined to close these 6 pull requests. In my view, most of these changes introduce errors or don't improve anything.

**Comment by b-mueller** [comment]
Guys, I'll merge the changes and take over editing of this chapter.

**Pull request merged**